examples/jf-control-app/commands/pairing/PairingCommand.cpp

```diff
@@ -40,9 +40,18 @@ using namespace ::chip::Controller;
 CHIP_ERROR PairingCommand::RunCommand()
 {
     CurrentCommissioner().RegisterPairingDelegate(this);
+    /* TODO: if JFA is onboarded get the administrator CAT initial version from JF_DS@GroupList (through RPC)
```

Comment:
What is "JF_DS@GroupList" ?

Reply:
Joint Fabric Datastore, GroupList attribute: https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/service_device_management/JointFabricDatastoreCluster.adoc?plain=1#L629